marketdata/entities.go

```diff
@@ -208,6 +209,11 @@ type CryptoSnapshot struct {
 	PrevDailyBar *CryptoBar   `json:"prevDailyBar"`
 }
 
+type CryptoPerpsTrade CryptoTrade
```

Comment:
`CryptoPerpsTrade` -> `CryptoPerpTrade`. Same for all other types and methods, where "perp" is used as an adjective before a noun.

Reply:
Done.